tests/run_test.go

```diff
@@ -68,20 +80,31 @@ func verifySnapshotCreated(snapName string) bool {
 	return true
 }
 
-func createSnapshot(pvcName, snapName string) {
+func createSnapshot(pvcName, snapName string, sized bool) {
```

Comment:
This size parameter was a bit confusing to figure out. Suggest you simply pass the const yaml var or the classname.

Reply:
> This size parameter was a bit confusing to figure out. Suggest you simply pass the const yaml var or the classname.

Passing the yaml name var now.